sky/utils/common_utils.py

```diff
@@ -117,7 +118,7 @@ def _base36_encode(num: int) -> str:
     return _base36_encode(int_value)
 
 
-def make_cluster_name_on_cloud(cluster_name: str,
+def make_cluster_name_on_cloud(local_cluster_name: str,
```

Comment:
We used `display_name` in other places. It would be nice to keep them aligned.
https://github.com/skypilot-org/skypilot/blob/008b51dcf3b4c1990b3bb24b83a92a830c131123/sky/provision/provisioner.py#L43